src/components/SelectInstitution.tsx

```diff
@@ -21,46 +27,102 @@ const StyledInstitutionSearchContainer = styled.div`
   width: 30rem;
 `;
 
+const StyledProgress = styled(Progress)`
+  display: block;
+  margin: 1rem;
+`;
+
 interface SelectInstitutionProps {
   onSubmit: (values: FormikInstitutionUnit) => void;
   onClose?: () => void;
-  excludeAffiliationIds?: string[];
 }
 
-const SelectInstitution: FC<SelectInstitutionProps> = ({ onSubmit, onClose, excludeAffiliationIds }) => {
-  const { t } = useTranslation('profile');
+const SelectInstitution: FC<SelectInstitutionProps> = ({ onSubmit, onClose }) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const institutions = useSelector((store: RootStore) => store.institutions);
+  const [selectedInstitution, setSelectedInstitution] = useState<RecursiveInstitutionUnit[]>();
+  const [fetchingInstitutions, setFetchingInstitutions] = useState(false);
+  const [fetchingDepartment, setFetchingDepartment] = useState(false);
```

Comment:
vi har brukt `isLoading` eller `loading` som standard term for dette, men vet ikke om det har så mye å si om vi bruker fetching og loading om hverandre?

Reply:
ble sånn siden metodene heter `fetchInstitutions` etc, men enig i at vi heller kan holde oss til loading!